lib/auth_app_web/live/user_live/login.ex

```diff
@@ -18,32 +18,63 @@ defmodule AuthAppWeb.UserLive.Login do
       <.simple_form for={@form} id="login_form" action={~p"/users/log-in"} phx-update="ignore">
         <.input field={@form[:email]} type="email" label="Email" autocomplete="username" required />
         <.input
+          :if={@mode == :password}
           field={@form[:password]}
           type="password"
           label="Password"
           autocomplete="current-password"
-          required
         />
-
         <:actions>
           <.input field={@form[:remember_me]} type="checkbox" label="Keep me logged in" />
-          <.link href={~p"/users/reset-password"} class="text-sm font-semibold">
-            Forgot your password?
-          </.link>
         </:actions>
-        <:actions>
+        <%!-- TODO: too many action slots necessary for my liking --%>
```

Comment:
It may be better to just render as two distinct forms instead of several conditionals.